java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_1/MusicalInstruments.java

```diff
@@ -0,0 +1,40 @@
+package com.zoolatech.lecture3.tasks._1;
+
+abstract public class MusicalInstruments {
```

Comment:
1. usually, the access modifier is defined first: `public abstract`
2. prefer to use a singular form for class names (MusicalInstrument, without s)